packages/shared/src/lib/socialMedia.ts

```diff
@@ -0,0 +1,52 @@
+import { type ComponentType } from 'react';
+import { RedditIcon } from '../components/icons';
+import { TwitterIcon } from '../components/icons/Twitter';
+import type { IconProps } from '../components/Icon';
+
+export enum SocialIconType {
+  Reddit = 'reddit',
+  X = 'x',
+}
+
+export const socialIcon: Record<SocialIconType, ComponentType<IconProps>> = {
+  [SocialIconType.Reddit]: RedditIcon,
+  [SocialIconType.X]: TwitterIcon,
+};
+
+export const socialGradient = {
+  reddit: 'bg-gradient-to-r !from-[#FF4500] !to-[#FF6B00]',
```

Comment:
This is supposed to be custom I guess?

Reply:
@rebelchris Yes, this is the color for the reddit ones. The importants are because of this weird issue: https://dailydotdev.slack.com/archives/C02E2C3C13R/p1732285583763759 🥴